ansible/roles/master/tasks/ghReleasesInstall.yml

```diff
@@ -0,0 +1,42 @@
+---
+- name: download kubernetes from github
+  get_url: url={{ kubernetes_download_url }} dest={{ kubernetes_download_dir }}/{{ kubernetes_download_filename }}
+
+- name: unarchive kubernetes
+  unarchive: copy=no
+             src={{ kubernetes_download_dir }}/{{ kubernetes_download_filename }}
+             dest={{ kubernetes_download_dir }}
+
+- name: unarchive kubernetes-server
+  unarchive: copy=no
+             src={{ kubernetes_download_dir }}/kubernetes/server/kubernetes-server-linux-amd64.tar.gz
+             dest={{ kubernetes_download_dir }}
+
+- name: copy kubernetes master binaries
+  shell: cp {{ kubernetes_download_dir }}/kubernetes/server/bin/{{ item }} /usr/bin
+  args:
+    creates: "/usr/bin/{{ item }}"
```

Comment:
I did this, but there was a minor issue. In master role there's a task that adds a linux capability to `kube-apiserver` and adding capabilities to links is not possible. I don't know enough about capabilities and don't know what's the best way for this scenario.